### PR TITLE
[UTOPIA-866] Match disabled text field UI for MPO use only text area

### PR DESCRIPTION
**Pull request by JieunSon96**
## 🎯 Summary

  
[UTOPIA-866](https://apps.itsm.gov.bc.ca/jira/browse/UTOPIA-866)

- Added separate CSS file for disabled editor to change the background color

## 🔰 Checklist

- [x] I have read and agree with the following checklist and am following the guidelines in our [Code of Conduct](/CODE_OF_CONDUCT.md) document.

> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - I have consulted with the team if introducing a new dependency.
> - My changes generate no new warnings.

### CODE REVIEW

File: src/frontend/src/components/public/PIAFormTabs/collectionUseAndDisclosure/index.tsx [hunk 1]
```diff
@@ -20,6 +20,7 @@ import {
 import ViewComments from '../../../common/ViewComment';
 import { PiaSections } from '../../../../types/enums/pia-sections.enum';
 import { TextInputEnum } from '../../../../constant/constant';
+import '../../PIAFormTabs/collectionUseAndDisclosure/_collectionUseAndDisclosure.scss';
```
Comment (BradyMitch) [inline src/frontend/src/components/public/PIAFormTabs/collectionUseAndDisclosure/index.tsx:23]: instead of importing, add it to common.scss

**Comment by BradyMitch** [comment]
We also need to prevent non-MPO users from switching back to edit mode. Its preview by default, but Drafter can switch it to edit mode.

**Comment by JieunSon96** [comment]
> We also need to prevent non-MPO users from switching back to edit mode. Its preview by default, but Drafter can switch it to edit mode.

do you mean by clicking the right side of the w-md-editor-toolbar buttons?

**Review by NoorChasib — APPROVED** [review]

File: src/frontend/src/components/public/PIAFormTabs/collectionUseAndDisclosure/index.tsx [hunk 2]
```diff
@@ -90,6 +90,20 @@ const PIACollectionUseAndDisclosure = ({
...
+    if (isMPORole() === false) {
+      // If the current user does not have the MPORole,
+      // select button elements inside #collectionNoticeMPO .w-md-editor-toolbar li.
+      const buttons = document.querySelectorAll<HTMLButtonElement>(
+        '#collectionNoticeMPO .w-md-editor-toolbar li > button',
+      );
+
+      // Iterate over all button elements and set the 'disabled' attribute to true.
+      buttons.forEach((button) => {
+        button.disabled = true;
```
Comment (antsand) [inline src/frontend/src/components/public/PIAFormTabs/collectionUseAndDisclosure/index.tsx:104]: is there not a disabled parameter we can pass in md-editor?

Comment (JieunSon96) [reply src/frontend/src/components/public/PIAFormTabs/collectionUseAndDisclosure/index.tsx:104]: > is there not a disabled parameter we can pass in md-editor?

no, currently if it is the preview mode (disabled to type) it disables the toolbar options on the left like bold, Italic bts but not the buttons on the right(Edit button,Live code, Preview Code etc) so I have written a code to disable it

**Review by antsand — APPROVED** [review]

**Pull request merged**